### ISSUE TITLE
Ability to hide videos in a collection

### ISSUE THREAD

**Issue by emmello** [mitodl/odl-video-service#198]
As an admin, I may want to delay viewing of a video.  So if I post a collection link on Stellar, I may not want to show all videos to make sure students come to class for live sessions.  

Example - 7.016 faculty delays posting the videos each week until the following Monday.  In DE, we had two HTML pages - one for admin access only and a second one with videos released for student viewing.  Having a box checkmark that an admin can click to turn on/off view for each lecture would allow them to control when they release content for viewing.

**Comment by pdpinch** [comment]
@mbertrand @emmello do you think this was fixed by https://github.com/mitodl/odl-video-service/pull/346 ?

**Comment by mbertrand** [comment]
Yes, but video permissions haven't been enabled on production yet (turned off until Youtube integration is complete).

**Comment by pdpinch** [comment]
This is blocked until we turn on public videos in production.
